common/postgresql-ng/values.yaml

```diff
@@ -41,23 +41,36 @@ config:
 
   random_page_cost: 1.1
 
+# Extra databases to create. A user with the same name MUST exist.
+# This is not checked by helm due to the to high complexity it would create in Helm's very limited templating engine.
+databases:
```

Comment:
This should either be called `.Values.extraDatabases`, or it should include the first database, and `.Values.postgresDatabase` should be removed. I'm leaning towards the latter: Since we're breaking backwards-compatibility anyway with the removal of `tableOwner`, I would prefer to have the interface be clean.

Reply:
I think we can only remove it if we solve the schema owner issue above. Otherwise we would need to magically take the first entry of databases for the schema owner which is not that great IMO.

Reply:
I think the best way forward is to split the init script to have a part that runs without `-d`, i.e. in the `postgres` database, where users/roles/databases are set up, and then have a part that runs with `-d`, i.e. inside the created database, once per database.